sql/resource_sql_global_config_test.go

```diff
@@ -110,8 +110,9 @@ func TestResourceSQLGlobalConfigDelete(t *testing.T) {
 				Method:   "PUT",
 				Resource: "/api/2.0/sql/config/warehouses",
 				ExpectedRequest: map[string]any{
-					"data_access_config": []any{},
-					"security_policy":    "DATA_ACCESS_CONTROL",
+					"data_access_config":        []any{},
+					"security_policy":           "DATA_ACCESS_CONTROL",
+					"enable_serverless_compute": false,
```

Comment:
This makes me just a bit nervous. Could we add one unit test to verify that if `enable_serverless_compute` is true in state that deleting it results in `"enable_serverless_compute": true` in the request?